conf/configure.fv3.wcoss2

```diff
@@ -0,0 +1,164 @@
+ifdef InNemsMakefile
+include $(TOP)/conf/configure.nems.NUOPC
```

Comment:
Why does this file not have any indentation?

Reply:
I copied it from configure.fv3.wcoss_dell_p3 and  configure.fv3.wcoss_cray, to be consistent with the others, I didn't change the format. However, I am OK to reformat it.